xcdat/regridder/accessor.py

```diff
@@ -306,13 +314,78 @@ def horizontal(
             )
 
         try:
-            regrid_tool = REGRID_TOOLS[tool]
+            regrid_tool = HORIZONTAL_REGRID_TOOLS[tool]
         except KeyError as e:
             raise ValueError(
-                f"Tool {e!s} does not exist, valid choices {list(REGRID_TOOLS)}"
+                f"Tool {e!s} does not exist, valid choices {list(HORIZONTAL_REGRID_TOOLS)}"
             )
 
         regridder = regrid_tool(self._ds, output_grid, **options)
         output_ds = regridder.horizontal(data_var, self._ds)
 
         return output_ds
+
+    def vertical(
```

Comment:
Based on #501, are we also considering calling this `vertical_xgcm` and not having a general wrapper with `tool="xgcm"`?

Reply:
That's correct, I'll address it in the next PR.